operators/src/machine_learning/onnx.rs

```diff
@@ -124,6 +130,14 @@ impl InitializedRasterOperator for InitializedOnnx {
     fn canonic_name(&self) -> CanonicOperatorName {
         self.name.clone()
     }
+
+    fn name(&self) -> &'static str {
+        Onnx::TYPE_NAME
+    }
+
+    fn path(&self) -> WorkflowOperatorPath {
+        self.path.clone()
+    }
```

Comment:
this whole thing looks like it should be covered by a macro

Reply:
I would not know how to achieve this in a way that actually saves code... We have no way to know which Operator an InitializedOperator belongs to so we can't access its name. Also each InitializedOperator is responsible for storing it's path, so we would only be able to save 3 lines in each operator but would not really gain anything and make it more unreadable.

I guess there would be a better way with an associated type and a InitializedOperatorBase that handles all the common stuff like saving the result descriptor and the name and the path but that refactoring would not gain us much I think and would cost several hours.

Reply:
You did 25x basically the same stuff. This is why it becomes Hard to write operators. Can be a different PR, tho.